Bottom: 1f3351ec97538dcb4670bcb4cc8874fc128412cd
Top:    00a36392278b8685288decbe2bc5ba9f5e032b96
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-11 02:06:18 +0100

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 1511e09..6987d2b 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
